src/api/GraphQuery/queryCRUD.ts

```diff
@@ -102,3 +102,21 @@ export interface ConvertQueryResponse {
   query: string;
   dialect: GraphQueryDialect;
 }
+
+export interface GetQueryStatsParams extends IDataSourceParams {
+  query: StructuredGraphQuery;
+  dialect: GraphQueryDialect.QUERY_BUILDER;
+}
+
+export interface GetQueryStatsResponse {
+  resultCount: {
```

Comment:
is it better to have accuracy and value in some dedicated interface ?
```
 resultCount: QueryStatsResultCount

export interface QueryStatsResultCount {
    accuracy: QueryStatsCountAccuracy;
    value: number;
}
```

Reply:
I'm not sure I see the benefit of introducing this interface. In any case, we can add it later if we change our minds.